areas/communication/src/AzureMcp.Communication/Commands/BaseCommunicationCommand.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.CommandLine;
+using System.CommandLine.Parsing;
+using System.Diagnostics.CodeAnalysis;
+using AzureMcp.Communication.Options;
+using AzureMcp.Core.Commands;
+
+namespace AzureMcp.Communication.Commands;
+
+public abstract class BaseCommunicationCommand<
+    [DynamicallyAccessedMembers(TrimAnnotations.CommandAnnotations)] TOptions>
+    : GlobalCommand<TOptions>
+    where TOptions : BaseCommunicationOptions, new()
+{
+    protected readonly Option<string> _connectionStringOption = CommunicationOptionDefinitions.ConnectionString;
```

Comment:
Are there secret values in this connection string?  If so, then get it from the mgmt plane using libs instead of exposing as a param.